src/test/java/stepdefinitions/StepDefinitions.java

```diff
@@ -56,10 +56,11 @@ public void iContactValori()throws Throwable{
     }
 
     @Then( "^the contact us form is available$" )
-    public void theContactUsFormIsAvailable() throws Throwable {
-        //TODO IMPLEMENT THIS
-        //Verify the contact form fields are available.
-        throw new PendingException();
+    public void theContactUsFormIsAvailable() {
+        //Added by FvZ
+        //So far only checks for H2-title of the contact form
+        ContactUsPage contactUsPage2 = new ContactUsPage(driver);
```

Comment:
Aangezien de scope van deze variabele enkel binnen deze methode valt, zou je deze gewoon contactUsPage kunnen noemen net als in de methode hierboven.